### ISSUE TITLE
MP3s

### ISSUE THREAD

**Issue by EthanMalakay** [yogstation13/yogstation#1595]
[Note]: make MP3s named NMP (Nanotrasen Music Player) (or YMP) with songs from lobby, and more added after :V

**Comment by Frostben** [comment]
Id like this, having an mp3 player would be nice for mah immursion

**Comment by EthanMalakay** [comment]
yeah i know right :V and it would help rp like, cant hear you man, im listenning my demn music :v

**Comment by enailz** [comment]
Add *scream and *laugh SFX while you're at it
